InteractiveHtmlBom/web/util.js

```diff
@@ -520,6 +521,8 @@ function initDefaults() {
   settings.markWhenChecked = readStorage("markWhenChecked") || "";
   populateMarkWhenCheckedOptions();
 
+  settings.kicadTextFormatting = config.kicad_text_formatting
```

Comment:
No need to have this in settings since user can't change it. You can refer directly to `config` in `render.js`.